internal/core/src/segcore/load_index_c.cpp

```diff
@@ -303,6 +303,10 @@ AppendIndexV2(CTraceContext c_trace, CLoadIndexInfo c_load_index_info) {
 
         auto config = milvus::index::ParseConfigFromIndexParams(
             load_index_info->index_params);
```

Comment:
set priority into the index config map, when IndexTranslator calling getCells method, the priority in the config will take effects